src/exercises/exerciseTypes/NextNavigation.js

```diff
@@ -31,14 +31,21 @@ export default function NextNavigation({
   const bookmarkToStudy = bookmarksToStudy[0];
   const exercise = "exercise";
   const [userIsCorrect, setUserIsCorrect] = useState();
-  const correctMessage = useState(random(correctStrings));
+  const [correctMessage, setCorrectMessage] = useState("");
 
   useEffect(() => {
     console.log("Message received: " + message);
```

Comment:
I left that comment in there for debugging, but I think we can remove it. The comments are also outdated, so I would also delete them.